src/authV2/azext_authV2/custom.py

```diff
@@ -452,11 +462,39 @@ def get_aad_settings(cmd, resource_group_name, name, slot=None):
 def update_aad_settings(cmd, resource_group_name, name, slot=None,  # pylint: disable=unused-argument
                         client_id=None, client_secret_setting_name=None,  # pylint: disable=unused-argument
                         issuer=None, allowed_token_audiences=None, client_secret=None,  # pylint: disable=unused-argument
+                        client_secret_certificate_thumbprint=None,  # pylint: disable=unused-argument
+                        client_secret_certificate_san=None,  # pylint: disable=unused-argument
+                        client_secret_certificate_issuer=None,  # pylint: disable=unused-argument
                         yes=False, tenant_id=None):    # pylint: disable=unused-argument
     if client_secret is not None and client_secret_setting_name is not None:
         raise CLIError('Usage Error: --client-secret and --client-secret-setting-name cannot both be '
                        'configured to non empty strings')
 
+    if client_secret_setting_name is not None and client_secret_certificate_thumbprint is not None:
+        raise CLIError('Usage Error: --client-secret-setting-name and --thumbprint cannot both be '
+                       'configured to non empty strings')
+
+    if client_secret is not None and client_secret_certificate_thumbprint is not None:
+        raise CLIError('Usage Error: --client-secret and --thumbprint cannot both be '
+                       'configured to non empty strings')
+
+    if client_secret is not None and client_secret_certificate_san is not None:
+        raise CLIError('Usage Error: --client-secret and --san cannot both be '
+                       'configured to non empty strings')
+
+    if client_secret_setting_name is not None and client_secret_certificate_san is not None:
+        raise CLIError('Usage Error: --client-secret-setting-name and --san cannot both be '
+                       'configured to non empty strings')
+
+    if client_secret_certificate_thumbprint is not None and client_secret_certificate_san is not None:
+        raise CLIError('Usage Error: --thumbprint and --san cannot both be '
+                       'configured to non empty strings')
+
+    if ((client_secret_certificate_san is not None and client_secret_certificate_issuer is None) or
+            (client_secret_certificate_san is None and client_secret_certificate_issuer is not None)):
+        raise CLIError('Usage Error: --san and --certificate-issuer must both be '
+                       'configured to non empty strings')
```

Comment:
Could you please use the specific error type from `azure.cli.core.azclierror` instead of `CLIError`?